test/Benchmarks/EventSourceBenchmarks.cs

```diff
@@ -20,7 +20,6 @@
 
 namespace OpenTelemetry.Benchmarks
 {
-    [MemoryDiagnoser]
```

Comment:
what is the benefit of providing this via command line? 
The cmd line is straightforward right now, as one can run it just like any dotnet run. I don't see any advantage of making the command line user needing to remember one more param.

Reply:
E.g. if I made a change that I know won't change any memory alloc, I could run the test in a much faster way 😃

Reply:
okay that seems to be a good reason.